### PR TITLE
[14.0][MIG] product_bom_revision

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

**Comment by emagdalenaC2i** [comment]
@patrickrwilson @tonygalmiche @bodedra @nikul-serpentcs and @rousseldenis Please, could you review this PR?

**Review by tonygalmiche — APPROVED** [review]
LGTM : Code review and functional review with runbot

image

File: product_bom_revision/views/mrp_view.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" ?>
+<odoo>
+    <!-- BOM revision form view -->
+    <record id="mrp_bom_form_view_revision" model="ir.ui.view">
+        <field name="name">mrp.bom.form.revision</field>
+        <field name="model">mrp.bom</field>
+        <field name="inherit_id" ref="mrp.mrp_bom_form_view" />
+        <field name="arch" type="xml">
+            <field name="product_id" position="after">
+                <field name="bom_revision" string="Revision" />
```
Comment (bodedra) [inline product_bom_revision/views/mrp_view.xml]:
```suggestion
                <field name="bom_revision" />
```

**Review by bodedra — APPROVED** [review]
LG :+1:

**Review by patrickrwilson — APPROVED** [review]
LGTM

**Comment by emagdalenaC2i** [comment]
@rousseldenis Please, I think this PR is ready to merge

File: product_bom_revision/views/mrp_view.xml (hunk 1 above)
Comment (rousseldenis) [reply product_bom_revision/views/mrp_view.xml]: @emagdalenaC2i

Comment (emagdalenaC2i) [reply product_bom_revision/views/mrp_view.xml]: Ok, done. Sorry, I missed that

**Comment by emagdalenaC2i** [comment]
@rousseldenis Please ;-)

**Comment by emagdalenaC2i** [comment]
@rousseldenis seems the bot needs a reset ;-)

**Comment by emagdalenaC2i** [comment]
@rousseldenis Are there any issue with the bot?

**Comment by rousseldenis** [comment]
> @rousseldenis Are there any issue with the bot?

Mmmh, maybe the issue Stephane told us about travis. Could you rebase to relaunch travis ?

**Comment by emagdalenaC2i** [comment]
@rousseldenis done

**Comment by emagdalenaC2i** [comment]
seems there is no change

**Comment by rousseldenis** [comment]
@sbidoul @simahawk It seems that no travis build is triggered when launching merge. Could you check ?

**Comment by sbidoul** [comment]
@rousseldenis can you try again ? For some reason travis had the flag "Build on push" disabled for this repo..

**Pull request merged**